src/services/photo/index.js

```diff
@@ -16,16 +16,35 @@ const CLOUD_BUCKET = 'staging.you-pin.appspot.com';
 
 const uploader = multer({
   inMemory: true,
-      fileSize: 5 * 1024 * 1024, // no larger than 5MB
-      rename: function (fieldname, filename) {
-        // generate a unique filename
-        return filename.replace(/\W+/g, '-').toLowerCase() + Date.now();
-      }
+  fileSize: 5 * 1024 * 1024, // no larger than 5MB
+  rename: function(fieldname, filename) {
+    // generate a unique filename
+    return filename.replace(/\W+/g, '-').toLowerCase() + Date.now();
+  }
 });
+
+function prepareMultipart(req, res, next) {
+  if (req.method.toLowerCase() === 'post') {
+    return uploader.single('image');
```

Comment:
We should return the following code otherwise it will be just hanging here in prepareMultipart and won't call the next middleware.

```return uploader.single('image')(req,res,next);```